crates/turborepo-auth/src/auth/logout.rs

```diff
@@ -1,16 +1,5 @@
-use anyhow::Result;
-use tracing::error;
-use turborepo_ui::{GREY, UI};
+use turborepo_ui::{cprintln, GREY, UI};
 
-pub fn logout<F>(ui: &UI, mut set_token: F) -> Result<()>
-where
-    F: FnMut() -> Result<()>,
-{
-    if let Err(err) = set_token() {
-        error!("could not logout. Something went wrong: {}", err);
-        return Err(err);
-    }
-
-    println!("{}", ui.apply(GREY.apply_to(">>> Logged out")));
-    Ok(())
+pub fn logout(ui: &UI) {
+    cprintln!(ui, GREY, ">>> Logged out");
```

Comment:
This does beg the question of whether this is even worth keeping as a function. cc @Zertsov @mehulkar

Reply:
It's fine to keep or remove. Auth / login changes may change this to have more functionality later, but agreed that at the moment it's kind of a lazy function.